js/getCountryCode.js

```diff
@@ -0,0 +1,258 @@
+function getCountryCode(countryName) {
```

Comment:
before this change the getCountryCode function was in a closure and therefore not globally accessible, now it is. I always try to not clutter the global namespace.

Reply:
The getCountryCode is data strings. I think to store a different entities apart is good idea. Global still works inside the same tab. Right? I know that copy with same name may break script if somebody try to change this somewhere else. Do you think there is high chance for this case or others issues?

Maybe code could be better.

Reply:
Sure, it would work. What we could do here is using our own namespace so the risk of having another function overwriting this one is very small. Something like

```
var FirefoxSprintCountries = {
  getCountryCody: function(countryName) {
    ..
  }
};
```

and then calling it with

```
FirefoxSprintCountries.getCountryCode('somecode');
```

What do you think?